libbeat/docs/newdashboards.asciidoc

```diff
@@ -51,13 +51,35 @@ $ ./scripts/import_dashboards -dir kibana/metricbeat
 $ ./scripts/import_dashboards -beat metricbeat -file metricbeat-dashboards-1.1.zip
 ----------------------------------------------------------------------
 
+- from the official zip archive available under http://download.elastic.co/:
+
+[source,shell]
+----------------------------------------------------------------------
+$ ./scripts/import_dashboards -beat metricbeat
+----------------------------------------------------------------------
+
 - from a zip archive available online:
 
 [source,shell]
 -----------------------
 $ scripts/import_dashboards -beat metricbeat -url https://github.com/monicasarbu/metricbeat-dashboards/archive/v1.1.zip
 -----------------------
 
```

Comment:
Do we need to have example for all config options? I think it is enough to document the most common once.